plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/reporting/VerifierErrorInterpreter.kt

```diff
@@ -37,6 +39,12 @@ class VerifierErrorInterpreter {
                 reportOn(source, PluginErrors.UNEXPECTED_RETURNED_VALUE, "null", context)
             error is PostconditionViolated && role is SourceRole.CallsInPlaceEffect ->
                 reportOn(source, PluginErrors.INVALID_INVOCATION_TYPE, role.paramSymbol, role.kind.asUserFriendlyMessage, context)
+            (error is PreconditionInCallFalse || error is AssertFailed) && role is SourceRole.ParamFunctionLeakageCheck -> {
+                val paramSymbol = error.extractInfoFromFunctionArgument(role.INDEX).unwrapOr<FirBasedSymbol<*>> {
+                    error("The function symbol is missing.")
+                }
+                reportOn(source, PluginErrors.LAMBDA_MAY_LEAK, paramSymbol!!, context)
```

Comment:
Would it be possible to strengthen the type of `unwrapOr` to ensure we don't need the `!!` here?